lib/publiccloud/instance.pm

```diff
@@ -79,14 +79,16 @@ sub run_ssh_command {
     if ($args{timeout} == 0) {
         # Run the command and don't wait for it - no output nor returncode here
         script_run($ssh_cmd, %args);
-    } elsif ($rc_only) {
+    }
```

Comment:
Please discard this.

Reply:
What do you mean with discard ?

Reply:
Are you sure this was not introduced by tidy?

Reply:
btw this change is the "original" style ...
I personally don't care if one or another.

Reply:
All of the origina `} elsif (...) {` and ```sub $func
{``` introduces tidy error. So I have to revise it for fixing the tidy error.